src/jit/codegenlinear.h

```diff
@@ -173,6 +173,14 @@ void genCodeForShiftLong(GenTreePtr tree);
 void genCodeForShiftRMW(GenTreeStoreInd* storeInd);
 #endif // _TARGET_XARCH_
 
+void genCodeForCast(GenTreeOp* tree);
+
+void genCodeForLclAddr(GenTree* tree);
+
+void genCodeForIndir(GenTreeIndir* tree);
+
+void genCodeForDiv(GenTreeOp* tree);
+
```

Comment:
I didn't see that; thanks. I'm going to use `genCodeForDivMod` for both archs, to be consistent, and just assert no MOD happens for arm64.